### PR TITLE
Start using real era_committees

### PR DESCRIPTION
**Pull request by adarowski**
Swap in the real `era_committee`s that my friends made for me.

@semanticart and @mikowitz — no action needed here, just wanted you to see how many lines of code this is saving. Not only is it removing 76 lines so far, it's also adding many more players to the list.

* [x] Show `era_committee` on player pages
* [ ] Exclude players with an `alt_hof` value from rake task
* [x] Show/hide lists beyond a certain point
* [ ] Don't show banned players as eligible

### CODE REVIEW

**Comment by semanticart** [comment]
😍

**Comment by adarowski** [comment]
I have a little bit of advice I need on this PR. Hopefully it's not too crazy.

When I made these lists by hand, I left out any players who are banned from baseball (since they won't appear on a ballot because of the ban). Doing this programmatically now adds them to the pool.

For an example, Pete Rose is appearing at the top of the Modern Baseball list for 2018. I added [this scope](https://github.com/adarowski/Hall-of-Stats/blob/9730185dd2eefd861ad751ae8b3e9a0cc73dd66b/app/models/player.rb#L113-L117) to collect the banned players. What I'd love to know is the best way to exclude these players from the list. Is this best to do in the rake task? Should I create a column in the database for the banned players and then just check against that in the rake task? Curious how you think I should handle this.

One other thing—the 2027 Today's Game ballot will also include players who belong to `eligible_in_2017`. Since these players are technically not yet eligible for the Hall, they are not covered in the rake task (and I also think they shouldn't be, so the rake task is fine). Basically, what I'd like to do [here](https://github.com/adarowski/Hall-of-Stats/blob/9730185dd2eefd861ad751ae8b3e9a0cc73dd66b/app/models/player.rb#L145) is scope this to be any players who fit the criteria on that line OR who are included in `eligible_in_2017`. I'm thinking that's probably pretty simple, but I just don't know the syntax.

❤️

File: app/views/players/show.html.haml [hunk 1]
```diff
@@ -61,6 +61,9 @@
         - if @player.alt_hof == 'nl'
           Major League
         player, but he is in the Hall of Fame as a #{@player.alt_hof_name}.
+    - if @bbwaa_2017_returning
```
Comment (adarowski) [inline app/views/players/show.html.haml]: @semanticart @mikowitz I'm using the `bbwaa_2017_returning` player scope here so I can show on a player page if he's returning to the ballot in 2017. This is always returning `true`, but basically what I want to ask is "is the player that I'm looking at included in the `bbwaa_2017_returning` group?" Is that relatively straight-forward?

Comment (semanticart) [inline app/views/players/show.html.haml]: Just pushed a commit that should cover this in the controller.

**Comment by semanticart** [comment]
> What I'd love to know is the best way to exclude these players from the list.

I added a `not_banned` scope so you can do things like ` Player.mb_era_2018.not_banned`

**Comment by semanticart** [comment]
> One other thing—the 2027 Today's Game ballot will also include players who belong to eligible_in_2017. Since these players are technically not yet eligible for the Hall, they are not covered in the rake task (and I also think they shouldn't be, so the rake task is fine). Basically, what I'd like to do here is scope this to be any players who fit the criteria on that line OR who are included in eligible_in_2017. I'm thinking that's probably pretty simple, but I just don't know the syntax.


If I understand what you're asking, then I think this should work:

```ruby
  scope :tg_era_2027, -> {
    where("ID IN (?) OR ID IN (?)",
      not_in_hof.where("era_committee = 'todays_game' AND last_year <= 2011 AND last_year >= 2009"),
      eligible_2017)
  }
```

That's not the most efficient way to do it but there's a small number of records here.

**Comment by adarowski** [comment]
> If I understand what you're asking, then I think this should work

That did the trick. 👍  Thanks!

**Comment by adarowski** [comment]
> I added a `not_banned` scope so you can do things like ` Player.mb_era_2018.not_banned`

Sweet. Will think about this some more since these players probably shouldn't have an `era_committee` value at all (now that I'm using that in other places). Can that scope be used within the rake task, per chance? I could also just add a column to the db if that's easier.

**Comment by semanticart** [comment]
It could be used in the task.

**Comment by adarowski** [comment]
> It could be used in the task.

Good to know. For now, I'm reconsidering listing those players as "eligible" (which would therefore remove them from the Hall of Stats). I've been considering this for a while—so that way both Halls pull from the same pool of eligible players.

**Pull request merged**